### PR TITLE
Beets

### LINKED ISSUE #502: Add beets
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Beets is currently missing, would be nice if beets could be added https://github.com/linuxserver/docker-beets (very useful tool for music library organisation).

**Describe alternatives you've considered**
Installing it locally.

**Additional context**
None

### PR DESCRIPTION
**Pull request by mcaron1234**
## Purpose

This closes #502 

## Approach

Add the required files

#### Learning

https://github.com/beetbox/beets
http://beets.io/

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/beets/beets.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# APPNICENAME=Beets
+# APPDESCRIPTION=music library manager and MusicBrainz tagger
```
Comment (nemchik) [inline compose/.apps/beets/beets.yml]: Please make the `m` in `music` upper case 😄

File: compose/.apps/beets/beets.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERCONFDIR}/beets:/config
+      - ${DOCKERSHAREDDIR}:/shared
+      - ${MEDIADIR_MUSIC}:/music
+      - ${DOWNLOADSDIR}:/downloads
```
Comment (nemchik) [inline compose/.apps/beets/beets.yml]: Please sort these alphabetically.

File: compose/.apps/beets/beets.armv7l.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+services:
+  beets:
+    image: linuxserver/beets
```
Comment (nemchik) [inline compose/.apps/beets/beets.armv7l.yml]: We also need the `aarch64` file as well.

**Comment by mcaron1234** [comment]
Fixed the doubled up empty line

**Comment by nemchik** [comment]
Awesome. I'll get the build to pass on Travis and get this merged asap.

**Comment by mcaron1234** [comment]
Great, this was harder than I was expecting, but that was probably because I did it all through ssh in the terminal. Now that I know how you like it done, I'd be happy to help with this type of stuff in the future. Feel free to assign me more tasks of this nature (I don't code, other than writing stupidly complex excel workbooks, lol) and I'll let you know if I have availability. I do a fair amount of traveling for work so sometimes it may take me longer than other times to get to something.

    On Wednesday, March 13, 2019, 3:38:36 PM PDT, Eric Nemchik <notifications@github.com> wrote:  
 
 
Awesome. I'll get the build to pass on Travis and get this merged asap.

**Comment by nemchik** [comment]
I live in SSH and terminal :grin: best thing to do as far as adding new apps is see what's in our issues list https://github.com/GhostWriters/DockSTARTer/issues

If I've already added it to the apps project https://github.com/GhostWriters/DockSTARTer/projects/8 then you're in the clear to go ahead and prepare a PR (preferably test the app as well).

If there's an issue I haven't labeled or added to a project it doesn't hurt to ping me on discord to confirm I don't have an issue with it being added (I usually don't, but sometimes I get behind on labeling).

If there's nothing you're interested in doing from our issues list it wouldn't hurt to have a look at https://www.linuxserver.io/our-images/ and add something we're missing.

**Pull request merged**